packages/drawer/src/ux-modal-service.ts

```diff
@@ -178,28 +202,47 @@ export class ModalService {
     const bindingContext: ModalBindingContext = {};
     const element = this.createDrawerElement(options, bindingContext);
     
-    element.innerHTML = `<compose view-model.ref=composeViewModel>`;
     if (!options.host || options.host === 'body') {
       options.host = document.body;
     } else if (typeof options.host === 'string') {
       options.host = document.querySelector(options.host) || document.body;
     }
     options.host.appendChild(element);
     let childView = this.templatingEngine.enhance({ element: element, bindingContext: bindingContext });
-    
-    if (options.viewModel && bindingContext.composeViewModel) {
-      bindingContext.composeViewModel.viewModel = options.viewModel;
-    }
-    if (options.view && bindingContext.composeViewModel) {
-      bindingContext.composeViewModel.view = options.view;
-    }
-    if (options.model && bindingContext.composeViewModel) {
-      bindingContext.composeViewModel.model = options.model;
-    }
 
+    let slot: ViewSlot;
     const controllers = (childView as any).controllers as Controller[];
-    
     const drawer: UxDrawer =  controllers[0].viewModel as UxDrawer;
+    try {
+      const view: any = controllers[0].view;
+      slot = new ViewSlot(view.slots['__au-default-slot-key__'].anchor, false);
```

Comment:
```suggestion
      slot = new ViewSlot(view.slots[ShadowDOM.defaultSlotKey].anchor, false);
```
We export the slot name here https://github.com/aurelia/templating/blob/5d84c7ee22df8fe92714b8abe99f5ef096542366/src/shadow-dom.js#L435